src/test/java/com/ays/assignment/model/dto/request/AssignmentListRequestBuilder.java

```diff
@@ -0,0 +1,72 @@
+package com.ays.assignment.model.dto.request;
+
+import com.ays.assignment.model.enums.AssignmentStatus;
+import com.ays.common.model.AysPaging;
+import com.ays.common.model.AysPagingBuilder;
+import com.ays.common.model.AysSorting;
+import com.ays.common.model.TestDataBuilder;
+
+import java.util.List;
+
+public class AssignmentListRequestBuilder extends TestDataBuilder<AssignmentListRequest> {
+
+    public AssignmentListRequestBuilder() {
+        super(AssignmentListRequest.class);
+    }
+
+    public AssignmentListRequestBuilder withValidValuesForStatus() {
+        return this
+                .withStatus()
+                .withPagination(new AysPagingBuilder().withValidValues().build())
+                .withSort(null);
+    }
+
+    public AssignmentListRequestBuilder withValidValuesForPhoneNumber() {
+        return this
+                .withPhoneNumber()
+                .withPagination(new AysPagingBuilder().withValidValues().build())
+                .withSort(null);
+    }
+
+    public AssignmentListRequestBuilder withValidValues() {
+        return this
+                .withPhoneNumberAndStatus()
+                .withPagination(new AysPagingBuilder().withValidValues().build())
+                .withSort(null);
+    }
+
+    public AssignmentListRequestBuilder withStatus() {
+
+        data.setFilter(new AssignmentListRequest.Filter(List.of(AssignmentStatus.AVAILABLE, AssignmentStatus.ASSIGNED)
+                , null));
+
+        return this;
+    }
```

Comment:
Bu tarz metotları tanımlamak yerine, spesifik metotlar tanımlayıp, değerleri `withValidFields()` metodunda vermemiz en doğrusu oluyor, fakat listeleme için filtreyi doğrudan test sınıfı içerisinde vermemiz en doğrusu oluyor